nixos/modules/services/mail/maddy.nix

```diff
@@ -220,20 +251,16 @@ in {
       '';
     };
 
-    users.users = optionalAttrs (cfg.user == "maddy") {
-      maddy = {
-        description = "Maddy service user";
+    users.users = optionalAttrs (cfg.user == name) {
+      ${name} = {
+        inherit uid;
```

Comment:
We have an explicit policy against using `uids` unless actually needed: https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md

Please use `isSystemUser` instead.

Reply:
Oh okay, good to know!